assets.md

```diff
@@ -90,6 +90,20 @@ Link: https://pypi.org/user/opentelemetry/
 
 ## Communication channels
 
+### CNCF Community Group
+
+Used by the End-User SIG to organize events (e.g. OTel Q&A, OTel in Practice, feedback sessions, etc.).
+
+CNCF are the owners of the group, so certain requests (e.g. adding organizers) need to be done via Service Desk.
+
+Link: https://community.cncf.io/opentelemetry/
```

Comment:
Pending discussion on this group possibly being renamed to _OpenTelemetry End-Users_.

Reply:
If the End-User SIG is OK in having other SIGs use it for their purposes in the future as well, I'm fine with having it as "OpenTelemetry".

Reply:
That's a good point. I assume these events would be public (as in, aimed at the general community) events regardless, so hopefully benefit from input/help from the End-User SIG to publicise them?

Reply:
After today's GC meeting, this conversation can be resolved and group name remains "OpenTelemetry"